include/linux/mmc/sdio_ids.h

```diff
@@ -98,6 +98,7 @@
 #define SDIO_DEVICE_ID_MARVELL_8997_BT		0x9142
 #define SDIO_DEVICE_ID_MARVELL_8977_WLAN	0x9145
 #define SDIO_DEVICE_ID_MARVELL_8977_BT		0x9146
+#define SDIO_DEVICE_ID_MARVELL_8978_WLAN	0x9159
```

Comment:
I would place the definition after `SDIO_DEVICE_ID_MARVELL_8987_BT` as the defines seem to be sorted by the value not the define/chip name.